docs/FAQ.md

```diff
@@ -79,25 +81,35 @@ For more about the closed source parts of VS Code, see [vscodium/vscodium](https
 
 ## How can I request a missing extension?
 
-Please open a new issue and select the `Extension request` template.
+To add an extension to Open-VSX, please see [open-vsx/publish-extensions](https://github.com/open-vsx/publish-extensions).
+We no longer plan to add new extensions to our legacy extension gallery.
 
 If an extension is not available or does not work, you can grab its VSIX from its Github releases or
 build it yourself. Then run the `Extensions: Install from VSIX` command in the Command Palette and
 point to the .vsix file.
 
-See below for installing an extension from the cli.
+You can install an extension on the CLI with:
+
+```bash
+# From the Coder extension marketplace
+code-server --install-extension ms-python.python
+
+# From a downloaded VSIX on the file system
+code-server --install-extension downloaded-ms-python.python.vsix
```

Comment:
It might be helpful to tell them how to do this from the command line. I know I struggled with it 😅 (it's even in my [dotfiles](https://github.com/jsjoeio/dotfiles#how-to-manually-install-vsx-files-for-code-server))